src/Pages/Profile/Styles/userProfile/UserProfile.styles.ts

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components"
+
+export const UserProfileSection = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 4rem 0 2rem;
+  padding: 4rem 0 0;
+  border-bottom: 1px solid;
+`
+export const UserProfileImageContainer = styled.div`
+  height: 16rem;
+  width: 16rem;
+  border-radius: 50%;
+  background-color: gray; //TODO: 컬러 변수로 등록하기?
```

Comment:
요부분도 마찬가지로 이렇게 바꿔보시면 좋을 것 같습니다! 😊
gray라는 색상은 theme.ts에 없어서 일단 sub로 제안드렸습니다!
```suggestion
background-color: ${({ theme }) => theme.colors.sub};
```